packages/create-waku/src/index.ts

```diff
@@ -162,9 +161,8 @@ Options:
 }
 
 async function notifyUpdate() {
-  // keep original require to avoid
-  //  bundling the whole package.json by `@vercel/ncc`
-  const packageJson = createRequire(import.meta.url)('../package.json');
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  const packageJson = require('../package.json');
```

Comment:
you can use `JSON.parse(fs.readFileSync(..))` if wanna remove require here

Reply:
Tried updating this from my phone. I think it's right but I'll need to correct the prettier issues from my laptop later haha

Reply:
I thought there was a motivation of preferring `require` to `readFileSync`? @himself65

Reply:
Nope, I just think it’s not a familiar way to use require in ESM

Reply:
https://github.com/dai-shi/waku/pull/978#issuecomment-2425461081